static/docs/api/frameworks/azcore/_bus_impl_8h.html

```diff
@@ -79,7 +79,7 @@
 <tr class="separator:"><td class="memSeparator" colspan="2">&#160;</td></tr>
 </table>
 <a name="details" id="details"></a><h2 class="groupheader">Detailed Description</h2>
-<div class="textblock"><p>Header file for internal EBus classes. For more information about EBuses, see <a class="el" href="class_a_z_1_1_e_bus.html">AZ::EBus</a> and <a class="el" href="struct_a_z_1_1_e_bus_traits.html">AZ::EBusTraits</a> in this guide and <a href="https://www.o3de.org/docs/user-guide/programming/ebus/">Event Bus documentation</a>. </p>
+<div class="textblock"><p>Header file for internal EBus classes. For more information about EBuses, see <a class="el" href="class_a_z_1_1_e_bus.html">AZ::EBus</a> and <a class="el" href="struct_a_z_1_1_e_bus_traits.html">AZ::EBusTraits</a> in this guide and <a href="https://www.o3de.org/docs/user-guide/programming/messaging/ebus/">Event Bus documentation</a>. </p>
```

Comment:
Since the links in the API Reference will be broken again the next time we generate these docs, are you planning to do a search and replace in the o3de/o3de repo to fix these links? If not, I'd create a GHI in that repo to track this work.

Reply:
I submitted a fix in the code: https://github.com/o3de/o3de/pull/14454